cmu_graphics/__init__.py

```diff
@@ -1,7 +1,13 @@
 import __main__
 import os
 
-main_directory = os.path.dirname(os.path.abspath(__main__.__file__))
+try:
```

Comment:
Instead of using a try/except here, can we use `hasattr` to check if `__main__` has a `__file__` attribute?